src/Core/Models/Domain/SendFile.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bit.Core.Models.Data;
+using Bit.Core.Models.View;
+
+namespace Bit.Core.Models.Domain
+{
+    public class SendFile : Domain
+    {
+        public string Id { get; set; }
+        public string Url { get; set; }
+        public string Size { get; set; }
+        public string SizeName { get; set; }
+        public CipherString FileName { get; set; }
+
+        public SendFile() { }
```

Comment:
Would this also need to call the `base()` constructor? and perhaps the loaded constructor calls `this()`

Reply:
It probably should. I think the compiler was willing to slap a default constructor in there for me, but I like the consistency.

I can see the `this()` call going both ways in the future. It depends on if we think things added to the default constructor should be called before the loaded one. Personally I wouldn't be expecting that and might do something wrong because of it.